distributed/http/scheduler/prometheus/core.py

```diff
@@ -49,9 +50,15 @@ def collect(self) -> Iterator[GaugeMetricFamily | CounterMetricFamily]:
             - len(self.server.saturated),
         )
         worker_states.add_metric(["saturated"], len(self.server.saturated))
+        paused_workers = len(
+            [w for w in self.server.workers.values() if w.status == Status.paused]
+        )
+        worker_states.add_metric(["paused"], paused_workers)
         worker_states.add_metric(
-            ["paused_or_retiring"], len(self.server.workers) - len(self.server.running)
+            ["retiring"],
```

Comment:
Out of scope: Technically, this definition is slightly off because there's a brief delay between registering a worker on the scheduler (with the worker being in the `init` state) and updating its state to `running` on the scheduler.